tools/build-webview

```diff
@@ -25,10 +25,31 @@ EOF
 
 # print error and die
 err() {
-    echo "$0:" $'\e[31merror:\e[0m' "$1" >&2
+    echo -e "$0:" $'\e[31merror:\e[0m' "$@" >&2
     exit 1;
 }
 
+# transform a possibly-relative path into an absolute path
+absolutify() {
+    case $(uname -a) in
+        Darwin*)
+            # The BSD-based readlink on OSX doesn't have an equivalent to the
+            # GNU `-f` option. Check for `greadlink` instead, and die if not
+            # found.
+            if type greadlink &> /dev/null; then
+                greadlink -m $1
+            else
+                err "\`greadlink\` not found; please install GNU coreutils: " \
+                    "\n   brew install coreutils"
+            fi
+        ;;
```

Comment:
Take a look at tools/lib/ensure-coreutils.sh :smile: 

Given the insight that this is not a problem in the first place on Windows, I'm happy to see coreutils become a requirement for normal development (cf the comment in that file). Probably most people who clone this repo on macOS have it installed somewhere anyway, via homebrew or otherwise.

Reply:
I did see that! Those are maintainer-only scripts, as you note – most contributors won't be mucking about in there to be confused by it – so I'm content to leave cleaning it out for another day and another PR.

(Also I've only checked `readlink`; if you need any other utilities from `coreutils`, they'll have to be verified separately.)